src/frontend/src/icp/services/icrc-custom-tokens.services.ts

```diff
@@ -28,26 +28,24 @@ export const buildIndexedIcrcCustomTokens = (): Record<
  */
 export const buildIcrcCustomTokens = (): IcTokenWithoutIdExtended[] => {
```

Comment:
Would it make sense to rename this? I find it rather confusing that we're talking about customTokens here (and in the filename)

Reply:
Ah, saw your comment in the other pr, all good